apiary.apib

```diff
@@ -1176,8 +1201,10 @@ There are a lot of filtering capabilities for this report. Most of the columns r
 - `_links` (object)
     - `creatives_export`
         - href: https://affiliate.api.diablomedia.com/v1/campaigns/1/creatives_export?publisher_id=10000 (string)
-    - suppression
+    - `suppression`
```

Comment:
Not sure how pertinent this change was, I noticed creatives_export had it above so added to suppression and optizmoaccess.

Reply:
I think it's only important if the field name contains characters that might break the parsing, so shouldn't be harmful to add.